applications/asset_tracker_v2/doc/debug_module.rst

```diff
@@ -38,6 +38,19 @@ This enables the application to be able to collect coredump data before a reboot
 To enable Memfault, you must include the :file:`../overlay-memfault.conf` when building the application.
 To get started with Memfault, see :ref:`using_memfault`.
 
+.. _asset_tracker_v2_ext_transport:
+
+Custom transport
+----------------
+
+The data that is collected from the device can be routed through an custom transport to the Memfault cloud instead of using Memfault's own HTTPS.
+To do this, enable the :ref:`CONFIG_DEBUG_MODULE_MEMFAULT_USE_EXTERNAL_TRANSPORT <CONFIG_DEBUG_MODULE_MEMFAULT_USE_EXTERNAL_TRANSPORT>` option.
+If this option is enabled, the debug module forwards captured Memfault data through the :c:enumerator:`DEBUG_EVT_MEMFAULT_DATA_READY` event.
+The data is forwarded through the configured cloud, for example, through AWS IoT, Azure IoT Hub, or `nRF Cloud`_, to Memfault cloud.
```

Comment:
I think this sentence can be removed. Its more or less a duplicate of

`The data that is collected from the device can be routed through an custom transport to the Memfault cloud instead of using Memfault's own HTTPS.`